.travis.yml

```diff
@@ -38,9 +37,8 @@ before_deploy:
 deploy:
   provider: releases
   api_key:
-    secure: ebvFFPdM4G2Rn9zc1m9KvMBN8bWsBh6rlRhEVSD3s+1vdxOSzhtsjZq3VWjh+eHT+QsvSnQZuL20Cq5YO31WraE/26a7hyx20JoFEEReab9LduGG26MJRapj400O982P/V1IFhMvzZEXB8uVHfpm4h0tVuOkECMGTyA1C0ztyOorjJBbC7kUzTZD+IAWWvIN07bxLgBLvbRzhXwlvJRkoC6GKcu63LA+8hVcL6j06fYFqmN3iY8AZVNwCyvRmA3bi92UUSFFtRfiAjw09gN1Z7pRJCKNlu9dSmZvFWqHyDCCFlwGzoBZPexVEAo2Mi6gOX/FDawBLovqS6CKYiQRfnDQHir+JHTwze47opsSn+bUrF8d6Dkq3enqDlVmy2JZiuo1xluRNhJc5cufg7HoN2fH92oGpo4d2989+NMhqsQTwU7OawbSag045jeL+YX+bOrlcmD/c+qRTFwQYFDc9fOi2UrFDD0Mx9+lAY8jfiLO8d4vB+4FEY/pF6+F4dCCSj8E3Qi6jcNQqhq5Kbx92JVEUopdzJlD0m2FoBPe23MkFhCV0mpCmbqyEeuTg00fbdQvijW6zHmNgtHAePxnE/Dua9txRoD8C3idTvBMNwZh78Het4Ltssb3p+lNWoKH3zgAB/OC3l0FVPpVJLbLTey/8cA/gSSJ8RIOJRwtCEQ=
+    secure: KmAvC/VoofISXnlzcfP3PDdVD4qNuoXHrf1g/6mPbRRqY1k4wET8GDaLBWsAcjY6hibqSmOrOjDV8TVzIEkKa5rfkjnh7GWPe5na4qZVvkmvmx8fHNWy6uFa0KKJxB8FG0+uPj8wtnIWutVYdWk1JTTBagRZ4l5UCm4catl4Yb3Avm4uYd3LIKRRXXmtgcc/B7E3xYQCf0tBAnx4qD4GNOs13B6h9m8jF7XZPTkRxk+vnpYB6rdlHLLKkNQ2PEP5QAdik6+eFCzTwtoP/NzOcdtCrl8aaGBJvpHGA8V+aZ/BvCn0YFfM1uW1IKp2dMDYYO0zUrWLDKT9IcG1oXh2uVduVEbMC48BVsOxIBs7NzMkXkiW/3/P3EtZ+0p/OWZr9g0CnS6sLeyGc0mLx/Tyl92yRiO44k+2AJFQSArEDn5w3+vuMQgsiK9tv0EdefPDyRzPdxarLFz3PSWrI360fZZmsUBaNuU5tMgUwOFFNrsSPjLsPUcI25BbaM+5WqFObn/bYNiUPuMpFM8X2sJT81OEQ9+gmkwUx2U0IgLUqpmcLOEncWi0NMf2bPl6KssP0NwJmwpV0u1SP0iEVoNnALNPrrsqYIkxkn7pvmlZKEcufz/7/posMA0umFiY1ABSn0aVISDJfZeXiqXRBAhIt1dPpGVvfxRc6zCA+fgl1jE=
   file: "./cfn-guard-$TRAVIS_OS_NAME-$TRAVIS_TAG.tar.gz"
   on:
-    repo: aws-cloudformation/cloudformation-guard
-    tags: true
+    branch: apikey-update
```

Comment:
shouldn't leave this branch name

Reply:
thought i removed that from testing. updated.